include/seqan3/io/alignment_file/detail.hpp

```diff
@@ -209,11 +229,9 @@ std::string get_cigar_string(ref_seq_type && ref_seq,
  *
  * \details
  *
- * \attention Note that CIGAR elements (respectively by their
- *            CIGAR operation) are always related to on one of the two sequences
- *            in a pairwise alignment. In this case, the resulting cigar_vector
- *            is based on sequence at the second position of the \p alignment pair,
- *            namely the query sequence.
+ * \attention Note that CIGAR elements (respectively by their CIGAR operation) are always related to one of the
+ *            two sequences in a pairwise alignment. In this case, the resulting cigar_vector is based on sequence
```

Comment:
I don't understand this. What information do you like to express?

Reply:
That the cigar string is a relative thing... If it says "deletion of 2 bases", it means a deletion in the query not the reference, so you need to aware which is the reference sequence and which is the query sequence. 

In the case of `map_aligned_values_to_cigar_op` it is query_char and reference_char.

Reply:
I don't know. I find the description a bit complicated. I proposed some alternative description in a comment above. What do you think?